src/common/Parameter.cpp

```diff
@@ -1496,8 +1499,9 @@ void Parameter::set_type(int ctrltype)
         displayInfo.customFeatures = ParamDisplayFeatures::kAllowsModulationsInNotesAndCents;
         break;
 
-    case ct_freq_shift:
+    case ct_freq_fm2_offset:
         displayType = LinearWithScale;
+    case ct_freq_shift:
```

Comment:
Why did this line move below the assignment?

Reply:
Oh that's just a flub I guess...